src/v2/spec.yaml

```diff
@@ -1317,6 +1317,48 @@ components:
           example: true
         action:
           $ref: "#/components/schemas/ValidatedFrameAction"
+    FramesResponse:
```

Comment:
rename this to `ValidateFrameListFrameResponse` and `ValidateFrameAnalyticsTotalInteractorsResponse` and so on...